Documentation/getting-started.md

```diff
@@ -1,26 +1,15 @@
 ## Getting started
 
+## Before Starting
+The Media for Edge Network extension has specific configuration requirements for including the Media Collection Details field group in the XDM schema, enabling Media Analytics in a datastream, and installing the Adobe Streaming Media for Edge Network extension in a Tag mobile property. For detailed information refer to the [Configure and Setup Adobe Streaming Media for Edge Network](https://developer.adobe.com/client-sdks/documentation/media-for-edge-network/#configure-and-setup-adobe-streaming-media-for-edge-network).
```

Comment:
- The Streaming Media for ...
- We can remove the details about the tag property configuration which are covered below, reading: "... has specific configuration requirements for including the Media Collection Details field group in the XDM schema and enabling Media Analytics in your datastream configuration. For more details, please refer to this guide to [configure and setup Adobe Streaming Media for  Edge Network](https://developer.adobe.com/client-sdks/documentation/media-for-edge-network/#configure-and-setup-adobe-streaming-media-for-edge-network).